dvc/main.py

```diff
@@ -37,7 +38,10 @@ def main(argv=None):
             logger.setLevel(logging.DEBUG)
 
         cmd = args.func(args)
-        ret = cmd.run_cmd()
+        ret = cmd.run()
+    except LockError:
+        logger.exception("failed to lock before running a command")
```

Comment:
What does it mean for a user? Can we make a message less cryptic? Explain what's actually happening and how it can be resolved

Reply:
@shcheklein Exception contains a more descriptive message already:
```
                "cannot perform the cmd since DVC is busy and "
                "locked. Please retry the cmd later."          
```

Reply:
It would look just like it did before:
```
ERROR: failed to lock before running a command - cannot perform the cmd since DVC is busy and locked. Please retry the cmd later.

Having any troubles?. Hit us up at https://dvc.org/support, we are always happy to help!
```